src/dask_awkward/lib/io/io.py

```diff
@@ -504,29 +496,104 @@ def __call__(self, packed_arg):
         )
 
 
-def return_empty_on_raise(
-    fn: Callable,
-    allowed_exceptions: tuple[type[BaseException], ...],
-    backend: BackendT,
-) -> Callable:
-    @functools.wraps(fn)
-    def wrapped(*args, **kwargs):
+def default_report_success(*args: Any, **kwargs: Any) -> ak.Array:
+    return ak.Array(
+        [
+            {
+                "args": [],
+                "kwargs": [],
```

Comment:
We probably want to ensure that this matches the type of the failure. Given that we know this ahead of time, we can express the default report as a form and use `ak.Array(form.length_zero_array(highlevel=False))`